### DISCUSSION TITLE
Livewire Sortable using defer modifier

### DISCUSSION THREAD

**Discussion by aspyropoulos** [livewire/livewire#3818] [1. Help] [answered]
I am using the [livewire sortable package](https://github.com/livewire/sortable) and it works fine out of the box. However, when I am trying with **defer** modifier it does not work. I want to first sort all of my images and when I click the save button to send the sorted order to the server. This is to avoid many HTTP requests to server.

wire:sortable.**defer**="updateTaskOrder" // This is not working

Is the **defer** modifier working? If not is it possible to achieve this behavior somehow?

**Comment by joshhanley** [comment, marked as answer]
@aspyropoulos the sortable package doesn't have support for the `defer` modifier, as it's an action (like `wire:click`) not an input update, as such it can't be deferred.

Hope this helps!
